testing/stargazer/APKBUILD

```diff
@@ -0,0 +1,40 @@
+# Contributor: Alexander Georgievskiy <galeksandrp@gmail.com>
+# Maintainer:
+pkgname=stargazer
+pkgname2=stg
+pkgver=2.408
+pkgrel=0
+pkgdesc="A billing system for small home and office networks"
+url="http://www.$pkgname2.dp.ua"
+arch="all"
+license="GPL2+"
+depends=""
+makedepends="expat-dev linux-headers"
+install=""
+subpackages=""
+source="http://www.$pkgname2.dp.ua/download/server/$pkgver/$pkgname2-$pkgver.tar.gz"
+builddir="$srcdir/$pkgname2-$pkgver/projects/$pkgname"
+options="!check"
+
+prepare() {
```

Comment:
Please call `default_prepare || return 1` at the beginning of custom `prepare` function.